### PR TITLE
Add 'sign up reason' section to the form

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #155 
screenshot from 2017-08-22 09 46 45
screenshot from 2017-08-22 09 47 03

### CODE REVIEW

File: database/startup_schema.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ const startupSchema = mongoose.Schema({
   startupSector: {type: String, required: true},
   email: {type: String, required: true},
   fundingStage: {type: String, required: true},
-  startupWebsite: {type: String, required: true },
+  startupWebsite: {type: String, required: true},
+  signUpReason: {type: String, required: true}
```
Comment (Karyum) [inline database/startup_schema.js]: Is it actually required thing ?

**Comment by Suuuuha** [comment]
@esraajb what do you think about the 'reason to signing up' title for this section? should we add a new filter to the results page as well ?

**Comment by Karyum** [comment]
@Suuuuha If i can remember correctly from the last standup that we had with @esraajb  we wanted those 'reasons' for ourself (or for UKITH), the user doesn't need to see them anywhere else other than the form

**Comment by Suuuuha** [comment]
@Karyum Travis is passing now you can review it again :)

**Comment by Karyum** [comment]
@Suuuuha https://github.com/UKITH/Arabs-Startups/pull/166#discussion_r134400572

File: database/startup_schema.js (hunk 1 above)
Comment (Suuuuha) [inline database/startup_schema.js]: I think if you need it for us as you said in your other comment, it should be requierd, no?
I'm happy to change it if you and @esraajb think it shouldn't.

**Comment by esraajb** [comment]
im opening an issue about the 'other' option. It would be better to have a text box shown when 'other' is clicked

**Pull request merged**